types/src/front/user.ts

```diff
@@ -6,20 +6,22 @@ import { WhitelistableFeature } from "./feature_flags";
 
 export type WorkspaceSegmentationType = "interesting" | null;
 
-export const RoleSchema = t.union([
-  t.literal("admin"),
-  t.literal("builder"),
-  t.literal("user"),
-  t.literal("none"),
-]);
+export const ROLES = ["admin", "builder", "user", "none"] as const;
+export const ACTIVE_ROLES = ["admin", "builder", "user"] as const;
+
+function keyObject<T extends readonly string[]>(
```

Comment:
Maybe we could simplify by doing something as here? https://github.com/dust-tt/dust/blob/main/types/src/front/feature_flags.ts

Reply:
It would have been an idea, but here the goal was to use io-ts without repeating the array, which requires to use this AFAICT

Reply:
Ah yes indeed that's because you want iots schema and type! 
I'm sure there's a way but I'm no expert and I don't think it's worth spending too much time on this 👍🏻